modules/nf-core/longphase/haplotag/tests/main.nf.test

```diff
@@ -0,0 +1,202 @@
+nextflow_process {
+
+    name "Test Process LONGPHASE_HAPLOTAG"
+    script "../main.nf"
+    process "LONGPHASE_HAPLOTAG"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "longphase"
+    tag "longphase/haplotag"
+
+    test("[ bam, bai, snps, [], [] ], fasta, fai") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/nanopore/bam/test.sorted.bam', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/nanopore/bam/test.sorted.bam.bai', checkIfExists: true),
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/gvcf/test.genome.vcf.gz', checkIfExists: true),
+                    [],
+                    []
+                ]
+                input[1] = [
+                    [ id:'reference' ],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta', checkIfExists: true)
+                ]
+                input[2] = [
+                    [ id:'reference' ],
+                    file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/genome.fasta.fai', checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.versions,
+                    bam(process.out.bam.get(0).get(1), stringency: 'silent').getHeader(),
+                    bam(process.out.bam.get(0).get(1), stringency: 'silent').getReadsMD5(),
```

Comment:
Awesome to see some `nft-bam` implementations :partying_face:

Reply:
What is the `stringency` actually doing here?

Reply:
I guess the BAM validation fails for those files

Reply:
Yes, I put some test files with readgroups not in header.. wasn't aware of it before using nft-bam :grimacing: